.travis.yml

```diff
@@ -31,15 +31,15 @@ before_install:
   # we add a symlink.
   - mkdir -p $TRAVIS_BUILD_DIR $GOPATH/src/github.com/letsencrypt
   - test ! -d $GOPATH/src/github.com/letsencrypt/boulder && ln -s $TRAVIS_BUILD_DIR $GOPATH/src/github.com/letsencrypt/boulder || true
-  - git clone https://www.github.com/letsencrypt/lets-encrypt-preview.git /tmp/letsencrypt
-  - cd /tmp/letsencrypt
+  - git clone https://www.github.com/letsencrypt/lets-encrypt-preview.git "$LETSENCRYPT_PATH"
+  - cd "$LETSENCRYPT_PATH"
   - sudo ./bootstrap/debian.sh
   - virtualenv --no-site-packages -p python2 ./venv
   - travis_retry ./venv/bin/pip install -r requirements.txt -e .
   - "cd -"
 
 env:
-  - LETSENCRYPT_VENV=/tmp/letsencrypt/venv
+  - LETSENCRYPT_PATH=/tmp/letsencrypt
```

Comment:
I wouldn't cache the whole `LETSENCRYPT_PATH` - I feel this would cause too many problems, latest repo and fresh venv is probably necessary not to get doomed too soon... but maybe we could cache downloaded PyPI packages to speed up the process... I will look into that.